### ISSUE TITLE
Can't save 4 channels tiff image properly

### ISSUE THREAD

**Issue by albertovitto** [cgohlke/tifffile#37]
**Describe the bug**
Hi,
I have a 400x800 4 channels tiff image. When I try to save it no error is raised but reading the saved image again will result in a completely different image (I'm using Photoshop 2020 as image editors), although the numpy arrays are equal. What am I doing wrong? 
Thank you

```python
from tifffile import tifffile
import numpy as np

tif = tifffile.imread("blue.tif") # ndarray (400, 800, 4)
tifffile.TiffWriter("blue_saved.tif").write(data=tif, dtype=np.uint8)

tif_saved = tifffile.imread("blue_saved.tif")

print(np.array_equal(tif , tif_saved) # True
```

**File**
[blue.zip](https://github.com/cgohlke/tifffile/files/5553184/blue.zip)

**Screenshots from Photoshop**

![image](https://user-images.githubusercontent.com/60853205/99387107-b54f0f80-28d3-11eb-87ac-55cb89245a05.png)

![immagine_2020-11-17_125333](https://user-images.githubusercontent.com/60853205/99387225-ea5b6200-28d3-11eb-8ef5-40c04e311d90.png)

**Environment**
 - platform : win-64
 - conda version [4.8.2]
 - Python [3.7.9]
 - tifffile [2020.10.1]
 - numpy [1.19.1]

**Comment by cgohlke** [comment]
The image in `blue.tif` is using a SEPARATE color space, probably CMYK. Tifffile doesn't currently support writing that colorspace but I'll add it to the next version so this will work: `imwrite('blue_saved.tif', image, photometric='SEPARATED')`

**Comment by albertovitto** [comment]
Thank you very much @cgohlke , keep it up with this great repo!

**Comment by cgohlke** [comment]
Should be fixed in v2020.11.18. Thanks for reporting the issue.
